plugins/linear-genome-view/src/LinearGenomeView/components/Header.tsx

```diff
@@ -108,35 +109,63 @@ const LinearGenomeViewHeader = observer(({ model }: { model: LGV }) => {
   const classes = useStyles()
   const theme = useTheme()
   const session = getSession(model)
-  const { coarseDynamicBlocks: contentBlocks, displayedRegions } = model
-
-  const setDisplayedRegion = useCallback(
-    (newRegionValue: string | undefined) => {
-      try {
-        if (newRegionValue) {
```

Comment:
can you explain a bit about the changes in this file, with the elimination of `useCallback`?

Reply:
Most of the changes done to this file including the elimination of the useCallback are to allow a further search to be made to the TextSearchManager to receive exact matches when a user has selected a prefix option from the dropdown. https://github.com/GMOD/jbrowse-components/blob/core_text_searching/plugins/linear-genome-view/src/LinearGenomeView/components/Header.tsx#L138-L168.